Remember last volume state when closing/reopening application

lukesnc opened the pull request.

jacksongoode commented:
This is good, but it might make sense to add some debounce so that we aren't writing to the config every time we scroll the slider.

lukesnc commented:
Forgive me if I'm misunderstanding the code, but I'm pretty sure what I added would work so that the write to config would only occur when the main window was being closed, which is when we'd save the config and close all windows anyway. I added debug statements on my end just to double check and it seems like the write only occurs on app exit

Pogodaanton commented:
Would this save the current volume in case of a crash, too?
